Create padStart.md (#3756)
Codecademy/docs: content/kotlin/concepts/strings/terms/padStart/padStart.md

---
Title: '.padStart()'

> Description: 'Pads the start of a character sequence with a string of the specified character and length.' 

Reviewer: ```suggestion
Description: 'Pads the start of a string with a specified character.' 
```

Subjects: 
  - 'Code Foundations'
  - 'Computer Science'
Tags: 
  - 'String'
  - 'Padding'
  - 'Kotlin'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

The **`.padStart()`** method pada the begining of the sequence with the specified fill character to reach the desired length, resulting in a new character sequence. **`.padStart()`** is also used for left-padding.

## Syntax

```pseudo

> String.padStart(length: Int, padChar: String): String

Reviewer: ```suggestion
String.padStart(length, padChar)
```

```

> 

Reviewer: ```suggestion

- 'String': The string to be modified.
```

> -`length`: The amount of padding to be added, in number of characters.

Reviewer: ```suggestion
-`length`: An integer that represents the total length of the string returned (with padding).
```

> `length`: The amount of padding to be added, in number of characters.
> `padChar`: Character to be used as padding, defaults to ' '.

Reviewer 2: ```suggestion
- `length`: The amount of padding to be added, in number of characters.
- `padChar`: Character to be used as padding, defaults to ' '.
```

> -`padChar`: Character to be used as padding, defaults to ' '.

Reviewer: ```suggestion
-`padChar`: The character to be used (enclosed by `''`) as padding, defaults to whitespace.
```

> ## Examples

Reviewer 2: ```suggestion
## Example
```
One change left.

> The example demonstrates the use of `.padStart()` to pad a string.

Reviewer: ```suggestion
The example below demonstrates the use of `.padStart()` to pad a string.
```

```kotlin
fun main() {
  val str = "codecademydocs"

  val str2 = str.padStart(14)
  println(str2)

>   val str3 = str.padStart(18, "-")

Reviewer: ```suggestion
  val str3 = str.padStart(18, '-')
```

> fun main {
>   val str = "codecademydocs"
> 
>   val str2 = str.padStart(3)
>   println(str2)
> 
>   val str3 = str.padStart(4, "-")
>   println(str3) 
>   }

Reviewer 2: ```suggestion
fun main() {
  val str = "codecademydocs"

  val str2 = str.padStart(14)
  println(str2)

  val str3 = str.padStart(18, '-')
  println(str3)
}
```
The print the below output you'll have to do these changes. The length of the string is 14. If you use 3 and 4 the string will be printed. No padding will be there.
